src/Symfony/Component/AccessToken/Credentials/Dsn.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AccessToken\Credentials;
+
+use Symfony\Component\AccessToken\Exception\InvalidArgumentException;
+
+/**
+ * @author Konstantin Myakshin <molodchick@gmail.com>
+ * @author Pierre Rineau <pierre.rineau@processus.org>
+ */
+final class Dsn
+{
+    public function __construct(
+        private readonly string $scheme,
+        private readonly string $host,
+        private readonly ?string $path = null,
+        private readonly ?string $user = null,
+        #[\SensitiveParameter]
+        private readonly ?string $password = null,
```

Comment:
```suggestion
        #[\SensitiveParameter] private readonly ?string $password = null,
```

Reply:
I'm strongly opposed to one liner when are too long. I'm one of the poeple working with a bigass screen, but splitting it into 2, 3, 4 sometime 5 editors.

Reply:
I agree, but that's how we do I it Symfony codebase, we love oneliners because it helps a lot to avoid merge conflicts

Reply:
OK, I'll rewrite it then, thanks.

Reply:
I just figured that I did it right at other places, I guess this is a glitch coming from the original copy/paste of the mailer component Dsn class.